derive/src/pyclass.rs

```diff
@@ -691,18 +772,25 @@ fn extract_impl_attrs(
                     bail_span!(path, "Unknown pyimpl attribute")
                 }
             }
+            NestedMeta::Meta(Meta::Path(path)) if path_eq(&path, "structseq_impl") => {
```

Comment:
it seems `pyclass` always matches `pyimpl` and `pystruct_sequence(with_impl)` always matches `pyimpl(structseq_impl)`. If they can't share same `pyimpl`, it can use a new name like `pyimpl_sequence_struct` or `pystruct_sequence_impl`. Did I miss some advantages from this sharing?